pytest/common_lib/shared/__init__.py

```diff
@@ -60,14 +62,14 @@ def sign_create_account_with_multiple_access_keys_tx(
         block_hash,
         creator_key.account_id,
         creator_key.decoded_pk(),
-        creator_key.decoded_sk()
+        creator_key.decoded_sk(),
     )
     return serialize_transaction(signed_tx)
 
 
 def start_neard_cluster_with_cleanup(
-        num_validators: int,
-        num_mpc_nodes: int,
+    num_validators: int,
+    num_mpc_nodes: int,
```

Comment:
I take it we don't have any python formatting in our CI? We should probably add a step to verify we follow some standard https://github.com/psf/black format.

Reply:
Lets add https://pypi.org/project/pylint/ (or some alternative) too!

Reply:
Happy to use yapf as well. That's what I used in past projects, but my understanding is that Black is the more popular one. No strong opinion on either.

Created an issue to do this: https://github.com/near/mpc/issues/593